src/plugins/intel_gpu/include/intel_gpu/primitives/binary_convolution.hpp

```diff
@@ -76,6 +76,20 @@ struct binary_convolution : public primitive_base<binary_convolution> {
         return seed;
     }
 
+    bool operator==(const primitive& rhs) const override {
+        if (!compare_common_params(rhs))
+            return false;
+
+        auto rhs_casted = downcast<const binary_convolution>(rhs);
+
+        return pad == rhs_casted.pad &&
+               stride == rhs_casted.stride &&
+               dilation == rhs_casted.dilation &&
+               groups == rhs_casted.groups &&
+               pad_value == rhs_casted.pad_value &&
+               weights.size() == weights.size();
```

Comment:
I think we can refactor these out as what @e-ddykim  done for proposal in this PR https://github.com/openvinotoolkit/openvino/pull/15335/files#diff-af2a6f9cbad55e0232a3e2689b9eb7702c7c33f0ae147c660fa28aa5618ed445, 
like, have a common binary dumper, use that binary for both hash and this comparator.

Reply:
But that kind of refactoring can be done in a separate work. Just let me leave a note.

Reply:
@yeonbok, it would be nice to have such unification. Will primitive serialization include fields like `second_output` of `ctc_greedy_decoder`? and other which are compared as `field_name.empty() == rhs_casted.field_name.empty()` in Roman's PR? 
And did you have a chance to check performance impact of such approach? Wouldn't it be slowly than simple filed by field comparison?